ee/clickhouse/client.py

```diff
@@ -137,11 +137,23 @@ def sync_execute(query, args=None, settings=None, with_column_types=False):
                 print()
                 print(format_sql(query, args))
```

Comment:
This is doing another substitution and formatting on its own. It's worth refactoring this and avoid needing to do formatting N times in one call.